src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,9 @@
-export const TodoList = () => {};
+import { Todo, TodoInfo } from '../TodoInfo';
+
+export const TodoList = ({ todos }: { todos: Todo[] }) => (
+  <div>
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings